_board/bast_wifi.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: download
+board_id: "bast_wifi"
+title: "bast_wifi Download"
+name: "bast_wifi"
+manufacturer: "Electronic Cats"
+board_url: "https://electroniccats.com/store/bast-wifi/"
+board_image: "bast_wifi.jpg"
+date_added: 2020-10-13
+
+features:
```

Comment:
Under features, you could add WiFi and anything else from the list at https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website/adding-to-downloads